src/app/js/formats/code/AdminComponent.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import translate from 'redux-polyglot/translate';
+import { polyglot as polyglotPropTypes } from '../../propTypes';
+
+const styles = {
+    container: {
+        display: 'inline-flex',
+    },
+    input: {
+        marginLeft: '1rem',
+    },
+};
+
```

Comment:
You need to export defaultArgs for field args to initialize properly when changing format
```js
export const defaultArgs = { languageToHighlight: '' },
```